src/main/java/com/eighthlight/fabrial/http/file/FileHttpResponder.java

```diff
@@ -187,13 +188,13 @@ private Response buildGetDirectoryResponse(Request request, ResponseBuilder buil
         fileController.getDirectoryContents(request.uri.getPath())
                       .stream()
                       .sorted()
-                      .map(p -> {
```

Comment:
@trptcolin The "HTML generation" has now been reduced to just a few lines. I've also `@Disabled` most of the (mostly obsolete) `AppAcceptanceTest` cases which should shave a good ~20s off the test execution time.